### DISCUSSION TITLE
How do I run a binary with the same name as a nushell command?

### DISCUSSION THREAD

**Discussion by MonstrousOgre** [nushell/nushell#13165] [Q&A] [answered]
Currently when I type a command starting with "http", it runs [the http command](https://www.nushell.sh/commands/docs/http.html). However, I have [HTTPie](https://httpie.io/) installed and the binary is called "http". I want to run that instead. Is there a way to do this without having to type out its full path?

**Comment by ghost** [comment, marked as answer]
you can use `^` to escape a command. example: `^http` or `^curl`.  
also: since custom definitions, etc can mess things up otherwise i would recommend always using it in scripts.
